readme.md

```diff
@@ -205,6 +205,7 @@
 - [AI Shader](https://github.com/keijiro/AIShader) - ChatGPT-powered shader generator for Unity.
 - [Translate GPT](https://github.com/ftp27/fastlane-plugin-translate_gpt) - A fastlane plugin that provides an action to translate localizable strings using ChatGPT.
 - [PandasAI](https://github.com/gventuri/pandas-ai) - Integrate ChatGPT capabilities into Pandas.
+- [doc-chatbot](https://github.com/dissorial/doc-chatbot) - Integrate ChatGPT capabilities into a document chatbot.
```

Comment:
The integration section is for things integrating with existing products. Yours does not do that.